database/migrations/2024_12_09_175613_add_index_on_created_at_for_comments.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
```

Comment:
Can you add the copyright disclaimer here (can be copy pasted from any other file)?